capstone.css

```diff
@@ -445,89 +445,89 @@ footer {
 
 @media only screen and (max-width: 768px) {
   .Hamburger {
-  display: inline-block;
-  height: 20px;
-  width: 20px;
+    display: inline-block;
+    height: 20px;
+    width: 20px;
   }
 
   .firsthead {
-  padding-bottom: 0;
+    padding-bottom: 0;
   }
 
   .firsthead h1 {
-  font-size: 30px;
+    font-size: 30px;
   }
 
   .program {
-  height: 930px;
+    height: 930px;
   }
 
   .logo {
-  display: none;
+    display: none;
   }
 
   .SocialBar {
- display: none;
+    display: none;
   }
 
   .navSmall {
- display: flex;
+    display: flex;
   }
 
   .All-Programs {
- grid-auto-flow: row;
- height: 900px;
- overflow-y: hidden;
+    grid-auto-flow: row;
+    height: 900px;
+    overflow-y: hidden;
   }
 
   .Aboutprogram p {
- height: 600px;
- overflow-y: hidden;
+    height: 600px;
+    overflow-y: hidden;
   }
 
   .Speaker {
- grid-template-columns: 1fr;
+    grid-template-columns: 1fr;
   }
 
   .NavBar {
- display: flex;
- flex-direction: column;
- height: 100%;
- width: 0;
- background-color: #272a31;
- color: white;
- position: fixed;
- top: 0;
- left: 0;
- transition: width 0.5s ease;
- overflow-x: hidden;
- z-index: 5;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 0;
+    background-color: #272a31;
+    color: white;
+    position: fixed;
+    top: 0;
+    left: 0;
+    transition: width 0.5s ease;
+    overflow-x: hidden;
+    z-index: 5;
```

Comment:
- [ ] By comparing your headline section with the one in the design provided, I can see that there are a lot of things that need to be improved. Please try to view the comparison in the screenshot below and I will try to add specific things to adjust 👇 👇 

**Given design**
![image](https://user-images.githubusercontent.com/90222110/191793513-7df74dce-0f15-4250-a632-d9d2b94eb194.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/191793752-616c8b5a-a409-4833-b2ae-c7b0ff5f2f11.png)


- [ ]  Starting with the navbar, there is the logo on the top-left side of the design. Please add the logo to your website too 👍 
- [ ] The background color of the headline section is supposed to be `light-grey` which differentiates it with the navbar. PLease implement it on your website.

- [ ] Please add the correct color to the active nav link. As it is in the design given, it is expected to have an orange color. Also, the styles on the about link can fool the visitors to think that they are on the About page while they are on the home page. So please correct it by aligning it with the design given
     
![image](https://user-images.githubusercontent.com/90222110/191795180-cf70b8c7-1fee-419a-b8d1-7db0b164f26c.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/191795475-7cb6e7b4-d4c8-4db3-b376-12c0d5345b2e.png)

- [ ] The heading text is supposed to be very big compared to others and also has this orange style. Please try to implement it in your design 🏃 
👇 
![image](https://user-images.githubusercontent.com/90222110/191795925-07f5a092-c7ec-4370-9c7e-91ff3634cbaa.png)

I hope that this is very helpful for your design improvements and I look forward to what you can achieve 😃 👍

Reply:
Yes, I will try to do my best.

Reply:
I did by increasing the font-size of .firsthead h1

Reply:
I made the color of the active nav element as required.
I made the headline background-color as "light-grey".